mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -397,27 +401,26 @@ def init_solver(self):
                 # self.solver.self_fields_verbosity = 2 if self.NONINTERAC else 0
 
     def init_conductors(self):
-        raise NotImplementedError(
-            "Diode conductors are not yet implemented in mewarpx")
         print('### Init Diode Conductors Setup ###')
         # # Create source conductors a.k.a the cathode
-        # self.cathode = warp.ZPlane(zcent=0., zsign=-1., voltage=self.V_CATHODE,
-        #                            name='cathode')
-        # self.cathode.WF = self.CATHODE_PHI
-        # self.cathode.temperature = self.CATHODE_TEMP
-        # warp.installconductor(self.cathode, dfill=warp.largepos)
+        self.cathode = assemblies.ZPlane(z=0., zsign=-1., V=self.V_CATHODE,
+                                    WF=self.CATHODE_PHI,
+                                    T=self.CATHODE_TEMP,
+                                   name='cathode')
 
         # # Create ground plate a.k.a the anode
         # # Subtract small z value for the anode plate not to land exactly on the
         # # grid
-        # self.anode_plane = warp.ZPlane(voltage=self.V_ANODE,
-        #                                zcent=self.D_CA - self.ANODE_OFFSET,
-        #                                name='anode_plane')
-        # self.anode_plane.WF = self.ANODE_PHI
-        # self.anode_plane.temperature = self.ANODE_TEMP
-        # warp.installconductor(self.anode_plane, dfill=warp.largepos)
+        self.anode_plane = assemblies.ZPlane(V=self.V_ANODE,
+                                       z=self.D_CA - self.ANODE_OFFSET,
```

Comment:
Actually let's just get rid of `ANODE_OFFSET` entirely in this code, I don't foresee using it for diodes when we always want to use the sim boundary now.